configure/executors/k8s/executor.ConfigMap.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: executor-config
+  labels:
+    app: executor
+    deploy: sourcegraph
+    sourcegraph-resource-requires: no-cluster-admin
+    app.kubernetes.io/component: executor
+# Refer to https://docs.sourcegraph.com/admin/executors/deploy_executors_binary#step-2-setup-environment-variables on how to populate these variables
+data:
+  EXECUTOR_USE_FIRECRACKER: "false"
+  EXECUTOR_KUBERNETES_PERSISTENCE_VOLUME_NAME: "sg-executor-pvc"
+  EXECUTOR_KUBERNETES_POD_AFFINITY: '[{"labelSelector": {"matchExpressions": [{"key": "app", "operator": "In", "values": ["executor"]}]}, "topologyKey": "kubernetes.io/hostname"}]'
+  # If Sourcegraph is not deployed in the `default` namespace, update this value
+  EXECUTOR_FRONTEND_URL: "http://sourcegraph-frontend.default.svc.cluster.local:30080"
+  EXECUTOR_MAXIMUM_NUM_JOBS: "8"
+  # Used configure which queues Executors will process.
+  # Can be "batches" or "codeintel"
+  # Either set this or EXECUTOR_QUEUE_NAMES.
+  EXECUTOR_QUEUE_NAME: ""
+  # Used configure which queues Executors will process.
+  # Can be "batches" or "codeintel" or "batches,codeintel"
+  # Either set this or EXECUTOR_QUEUE_NAME.
+  EXECUTOR_QUEUE_NAMES: "batches,codeintel"
+  EXECUTOR_KUBERNETES_RESOURCE_REQUEST_MEMORY: "5Gi"
+#  KUBERNETES_RUN_AS_USER: ""
+#  KUBERNETES_RUN_AS_GROUP: ""
```

Comment:
These will fail to parse during validation at startup time if they are provided as empty strings. Even though they are commented out, might be worth making them `-1` (which is the executor default)

Reply:
Good catch!